packages/amazonq/test/unit/codewhisperer/util/runtimeLanguageContext.test.ts

```diff
@@ -177,7 +177,6 @@ describe('runtimeLanguageContext', function () {
             'jsx',
             'kotlin',
             'php',
-            'plaintext',
```

Comment:
is tested here
https://github.com/aws/aws-toolkit-vscode/blob/master/packages/amazonq/test/unit/codewhisperer/util/runtimeLanguageContext.test.ts#L51